lib/KBox/src/tasks/ADCTask.cpp

```diff
@@ -33,11 +33,18 @@ void ADCTask::loop() {
   int bat1_adc = adc.analogRead(bat1_analog, ADC_0);
   int bat2_adc = adc.analogRead(bat2_analog, ADC_0);
   int bat3_adc = adc.analogRead(bat3_analog, ADC_0);
+    
+  //permits rudder sensor on bat3 input.  Eventually should move rudder sensor calcs to a separate task
```

Comment:
I was just taking a shortcut here for now.  I assume the better solution is to create a separate RudderSensor task and move the relevant code into that.